f5-distributed-cloud/assets/service_checks.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
This file can be removed since this integration does not include any service checks

Reply:
Hello, I  have committed the suggested updates.

Reply:
Hi @gregcoward, this file can be removed